interfaces/builtin/serial_port_test.go

```diff
@@ -463,14 +470,17 @@ func (s *SerialPortInterfaceSuite) TestConnectedPlugAppArmorSnippets(c *C) {
 	expectedSnippet7 := `/dev/ttyXRUSB0 rw,`
 	checkConnectedPlugSnippet(s.testPlugPort1, s.testSlot7, expectedSnippet7)
 
-	expectedSnippet8 := `/dev/tty[A-Z]*[0-9] rw,`
```

Comment:
@bergotorino 
```
koza: hey
koza: looking at #4819, can you tell me what the various test changes are for
koza: I was expecting just some new test chunk
koza: not alterations to existing tests
```

Reply:
There are new chunks but placed in a row. It has been convenient to put the in the "tty section" which was like "test-port-8" but then I realized that the "expectedSnippet8" is already taken by udev snippets. It feels like natural to have the numbers match therefore I have moved the existing entries one level down, i.e. 8->9; 9-10,... to make a room for this new one.

Reply:
+1